docs/user-guide/reading_writing_data.rst

```diff
@@ -105,6 +105,15 @@ You can now create the :py:class:`~hermes_core.timedata.TimeData` object,
     >>> from hermes_core.timedata import TimeData
     >>> timedata = TimeData(data=ts, meta=input_attrs)
 
+The :py:class:`~hermes_core.timedata.TimeData` object also accepts optional arguments for loading
+support data and non-record-varying (NRV) data. These arguments are required to each be a `dict`
```

Comment:
What is the difference between support data and NRV data?

Reply:
I would recommend we not use the term NRV (which even the SPDF folks don't seem to like). It is not clear without being explained.